src/index.ts

```diff
@@ -1,10 +1,32 @@
-let arreglo : number = 6;
-let vector1 : number [] = [1,3,7,9,9,5];
-let vector2 : number [] = [6,9,2,5,9,4];
-let sumaVector : number [] = new Array (arreglo);
-let indice : number;
-for (indice=0 ; indice < arreglo; indice ++) {
-  sumaVector[indice] = vector1[indice] + vector2[indice] ; 
-};
-
-console.log ("la suma es"  +sumaVector);
\ No newline at end of file
+let arregloDimension: number = 6;
+
+let vector1: number[] = new Array(arregloDimension);
+let numero1: number;
+for (let indice = 0; indice < arregloDimension; indice++) {
+  numero1 = Number(
+    prompt(
+      "Arreglo1: Indique el numero que desea incorporar en la posicion " +
+        indice
+    )
+  );
+  vector1[indice] = numero1;
+}
+
+let vector2: number[] = new Array(arregloDimension);
+let numero2: number;
+for (let indice = 0; indice < arregloDimension; indice++) {
+  numero2 = Number(
+    prompt(
+      "Arreglo1: Indique el numero que desea incorporar en la posicion " +
+        indice
+    )
+  );
+  vector2[indice] = numero2;
+}
+
+let sumaVector: number[] = new Array(arregloDimension);
+for (let indice = 0; indice < arregloDimension; indice++) {
+  sumaVector[indice] = vector1[indice] + vector2[indice];
+}
+
+console.log("la suma es " + sumaVector);
```

Comment:
La unica correccion es que los vectores no deben estar predefinidos, los valores de los vectores los tiene que ingresar el usuario, segun lo que dice la consigna.

Por eso agregue los dos for donde se pide al usuario que agregue los valores de cada vector, para despues hacer la suma.